archetypes/gwtp-basic/src/main/java/com/arcbees/gwtpbasic/GwtpBasic.gwt.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE module PUBLIC "-//Google Inc.//DTD Google Web Toolkit 2.7.0//EN"
+        "https://raw.githubusercontent.com/gwtproject/gwt/2.7.0/distro-source/core/src/gwt-module.dtd">
```

Comment:
Just a thought when I see this. I think the response type on the server should be changed for these types: 
This is what we get, `Resource interpreted as Document but transferred with MIME type application/xml-dtd: "http://www.gwtproject.org/doctype/2.7.0/gwt-module.dtd".` Idea complains about this. :)